src/PrestaShopBundle/Translation/PrestaShopTranslatorTrait.php

```diff
@@ -189,7 +185,6 @@ private function shouldFallbackToLegacyModuleTranslation($message, $domain, $tra
         return
             $message === $translated
             && 'Modules.' === substr($domain, 0, 8)
-            && substr_count($domain, '.') === 1
```

Comment:
this check is required because the first condition is met by 2 domains as stated in the PHPDocs